### DISCUSSION TITLE
Connecting on https://web3modal.com

### DISCUSSION THREAD

**Discussion by kandrukKV** [WalletConnect/walletconnect-monorepo#2462] [Developer Support] [answered]
Hi guys!

On the site https://web3modal.com/ there is a "Connected" button at the top.
After pressing it, a window appears where you can connect using mobile Metamask.

Where can I see sample this code? How did you manage to connect using metamasks?

**Comment by glitch-txs** [comment, marked as answer]
hi, this is web3modal 
https://docs.walletconnect.com/2.0/web3modal/react/installation
you can also find examples here: https://github.com/WalletConnect/web3modal-examples
